components/credentials/src/test/java/org/cloudfoundry/credhub/services/DefaultCredentialVersionDataServiceTest.java

```diff
@@ -317,6 +319,7 @@ public void delete_onACredentialName_deletesAllCredentialsWithTheName() {
   }
 
   @Test
+  @Transactional(propagation = Propagation.NEVER)
```

Comment:
I don't understand this. I read your commit message about it, but I still don't really get it. Does it force the test to run more serially or something?